susemanager-utils/susemanager-sls/modules/engines/mgr_events.py

```diff
@@ -204,7 +201,8 @@ def start(**config):
     '''
     Listen to events and write them to the Postgres database
     '''
-    io_loop = ZMQDefaultLoop.current()
+    io_loop = salt.ext.tornado.ioloop.IOLoop(make_current=False)
+    io_loop.make_current()
```

Comment:
Is this the same as the code below?

```suggestion
    io_loop = salt.ext.tornado.ioloop.IOLoop(make_current=True)
```

Reply:
I'm not 100% sure, but apparently the behavior might not be the same.

It seems creating the IOLoop instance with `make_current=True` might raise an exception when there is already another IOLoop: https://www.tornadoweb.org/en/stable/ioloop.html?tornado.ioloop.IOLoop#tornado.ioloop.IOLoop (bottom)

There is also this note for `IOLoop.make_current` method:

Makes this the IOLoop for the current thread.

*An IOLoop automatically becomes current for its thread when it is started, but it is sometimes useful to call make_current explicitly before starting the IOLoop, so that code run at startup time can find the right instance.*